### PR TITLE
Include labextension files in git

### PR DESCRIPTION
**Pull request by hainm**
#964 

Idea:
- Pregenerate the lab extension files and copy them to `nglview/labextension`. This is the same way we are doing for notebook extension (`nglview/static`).
- `pip install .` will create both notebook and lab extension folders (and `pip uninstall` will do the cleanup)

Why?
`pip install nglview` should work out of the box for both traditional notebook and jupyterlab (>= 3.0).

### CODE REVIEW

**Comment by hainm** [comment]
Uhm, filenames are random. We will end up having too many files:

```
nglview-js-widgets/labextension/static/
0de3b51742ed3ac61435875bccd8973b.png    421.04c3327cf7f16704c3b2.js             755.096b3c4de99557ca2b8c.js             a4c733ec4baef9ad3896d4e34a8a5448.png
133.0c12809866ed16ed02fe.js             421.04c3327cf7f16704c3b2.js.LICENSE.txt 755.096b3c4de99557ca2b8c.js.LICENSE.txt bf27228a7d3957983584fa7698121ea1.png
208a290102a4ada58a04de354a1354d7.png    534.eaec54f1d3737a7e98b9.js             776.5c1b8b0dcd384e09bac7.js             remoteEntry.283b03066e74058eed4d.js
325.0f8271db6262c6939811.js             568.ce222662ee73bd0a7f57.js             963.5cb06ad23bfa28bcdb1a.js             style.js
325.0f8271db6262c6939811.js.LICENSE.txt 73a1fd052c9d84c0ee0bea3ee85892ed.png    971364734f3b603e5d363a2634898b42.png
```

File: devtools/release.py [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 import subprocess
 import sys
+from pathlib import Path
+
+HERE = (Path(__file__).parent / '../').resolve().absolute()
```
Comment (robertodr) [inline devtools/release.py:7]:
```suggestion
HERE = (Path(__file__).parents[1]).resolve().absolute()
```

File: devtools/update_versions.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import json
+from pathlib import Path
+import sys
+from packaging.version import parse as parse_version
+
+version = sys.argv[1]
+HERE = (Path(__file__).parent / '../').resolve().absolute()
```
Comment (robertodr) [inline devtools/update_versions.py:7]:
```suggestion
HERE = (Path(__file__).parents[1]).resolve().absolute()
```

**Review by robertodr — COMMENTED** [review]
It looks good to me, but take my comment with a grain of salt. I am not an expert in jupyter extensions (notebook or lab) and for the JL3 update I followed the step-by-step instructions and no more...

File: devtools/release.py (hunk 1 above)
Comment (hainm) [reply devtools/release.py]: updated. thanks

File: devtools/update_versions.py (hunk 2 above)
Comment (hainm) [reply devtools/update_versions.py]: updated

**Pull request merged**